app/models/concerns/profile_hosts.rb

```diff
@@ -9,19 +9,8 @@ module ProfileHosts
     has_many :test_result_hosts, -> { distinct },
              through: :test_results, source: :host
     has_many :rule_results, through: :test_results
-    has_many :profile_hosts, dependent: :delete_all
```

Comment:
Sorry, I don't understand. Could you please elaborate? This is the profile -> profile_host relationship that must be removed if we remove the ProfileHost model.

Reply:
I've tried to point out that we should be cautious with migrations that do removals to avoid something like `ActiveModel::MissingAttributeError` (from RHICOMPL-1108). This should not be a problem for this case, as this is an association rather than a field. However, I don't know how it would behave. It could be tested by disabling spring, keeping our services running, doing the migrations, and checking that things work as expected.

Reply:
Yes, in this case, I don't think it should be an issue, since no active code (other than old data migrations) use `profile_hosts`.